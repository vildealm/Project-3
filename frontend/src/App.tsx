import React from 'react';
import logo from './logo.svg';
import './App.css';
import "reflect-metadata"; 

function App() {

  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <div className="pic-border">
          <h1> Hello World </h1>
        </div>
      </header>
    </div>
  );
}

export default App;
