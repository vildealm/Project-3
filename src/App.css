.header{
    color:rgba(255, 255, 255, 0.966);
    margin: auto;
    text-decoration: none;
    font-size: 1.5em; 
    line-height: 4;
    font-weight: lighter;
    font-family: 'Montserrat', sans-serif;
}