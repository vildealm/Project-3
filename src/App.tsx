import React from 'react';
import './App.css';
import { Dropdown } from './Dropdown/Dropdown';
import { Output } from './Components/List/Output/Output';
import { Search } from './Components/List/Search';
import { PersonState } from './reducers/reducer';
import { success, failure, begin, searchValue } from './actions/action';

const PERSON_API_URL = ``;


const App = (props) => {
  const fetchUrl = (url) => {
    props.begin();
    fetch(url)
      .then(response => response.json())
      .then(jsonResponse => {
        props.success(jsonResponse.docs);
      });
  }

  return (
    <div className="App">
      <header className="App-header">
        <h1> Visage Lexicon </h1>
        <Search />
        <Dropdown 
          loading={props.loading}
          movies={props.movies}
          errorMessage={props.errorMessage}
          />
        <Output />

      </header>
    </div>
  );
}

export default App;
