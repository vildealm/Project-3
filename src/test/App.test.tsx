import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import App from '../App';
import Modal from 'react-modal'



describe('App component', ()=> {
  it('renders the App-component', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
})

