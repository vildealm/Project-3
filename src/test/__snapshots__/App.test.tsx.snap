// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the App-component 1`] = `ShallowWrapper {}`;
